editions/tw5.com/tiddlers/community/resources/_file-backups_ Extension Firefox by pmario.tid

```diff
@@ -5,13 +5,13 @@ created: 20171116165500000
 delivery: Browser Extension
 description: Browser extension for Firefox
 method: save
-modified: 20210106151027036
+modified: 20221210215416509
```

Comment:
For minor changes like this I try to avoid changing the modified date so that the change doesn't show up in the "Recent" tab.